src/main/java/io/percy/appium/providers/AppAutomate.java

```diff
@@ -76,6 +81,39 @@ public void executePercyScreenshotEnd(String name, String percyScreenshotUrl, St
         }
     }
 
+    public String executePercyScreenshot() {
+        try {
+            if (AppPercy.getIsfullPageEnabled()) {
```

Comment:
we wanted a ENV var here just to test out the uploading via terminal and have it backwards compatible. if env var is set we use logic to hit new endpoint to take screenshot. otherwise use old method of driver.take_screenshot